### ISSUE TITLE
joins don't work when data.frame has only one column

### ISSUE THREAD

**Issue by dfalbel** [dgrtwo/fuzzyjoin#16]
See those examples:

```
library(dplyr)
library(ggplot2)
data(diamonds)
d <- data_frame(regex_name = c("^Idea", "mium", "Good"))
```

This works fine as diamonds as a `tibble`: 

```
diamonds %>%
  select(cut) %>%
  regex_left_join(d, by = c(cut = "regex_name"))
# A tibble: 53,940 x 2
         cut regex_name
       <ord>      <chr>
1      Ideal      ^Idea
2    Premium       mium
3       Good       Good
4    Premium       mium
```

This does not work:

```
diamonds %>%
  select(cut) %>%
  as.data.frame() %>%
  regex_left_join(d, by = c(cut = "regex_name"))
Error: cannot convert object to a data frame
```

**Comment by dfalbel** [comment]
OOops I'm not using Github version. This may be related to the CRAN version.
Just saw this: #13

**Comment by dgrtwo** [comment]
That's right, this was fixed by #13. Hoping to submit it to CRAN in next two weeks; until then please use GitHub version or convert to tbl_df first
